include/cpr/payload.h

```diff
@@ -12,9 +12,12 @@
 namespace cpr {
 
 struct Pair {
+    Pair(const std::string& p_key, const std::string& p_value)
+            : key(p_key), value(p_value) {}
     Pair(std::string&& p_key, std::string&& p_value)
             : key(std::move(p_key)), value(std::move(p_value)) {}
-    Pair(std::string&& p_key, const std::int32_t& p_value)
+    template<typename ValueType, typename std::enable_if<std::is_integral<ValueType>::value, bool>::type = true>
```

Comment:
Might just me, but why do we need to template this function?
I prefer it, if users have to convert their values to string by them selves to prevent unexpected behaviour.